### ISSUE TITLE
IAP is replaced by classic subscription flow on iPad (iOS 14.2)

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-mobile/mozilla-vpn-client#369]
#### Affected versions:
- 2.0.0;

#### Affected Platforms:
- iOS 14.2;

#### Prerequisites:
- Have an iPad;
- Have the VPN app installed;
- Have no FxA account;

#### Steps to reproduce:
1. Open the VPN app;
2. Tap “Get Started”;
3. Finalize authentication while signing up;
4. Observe the subscription screen;

#### Expected result:
-  IAP screen is displayed.

#### Actual result:
- The classic subscription flow is triggered.

#### Notes:
- If the account is already created, I cannot reproduce the issue.
- if we use a iPhone, we cannot reproduce the issue.
![IMG_20201209_161310](https://user-images.githubusercontent.com/15685960/101646765-9eb65700-3a40-11eb-80d1-2725c0b70c32.jpg)

**Comment by ValentinaPC** [comment]
For some reason, without doing something specific, this issue is no longer reproducible on my iPad.
The only thing I did different from this morning was: installing the prod build and checking the IAP issue in there=> all good. And after that , uninstalled the prod build, installed the beta build=> no IAP issue!

Not sure if this contribute to the issue not reproducing anymore.
I am a bit concern in this happening for real users, as we do not have the cause that triggered the issue.
 Closing.
